_posts/2022-10-24-ode-to-join.md

```diff
@@ -0,0 +1,70 @@
+It has already been a month since the 9th edition of the JOIN conference was concluded. 
+And even though it had been ten years since the first edition, the skipped year due to the pandemic will cause us to have an additional anniversary edition next year. 
+The aluminum edition will be a sight to see, and it would be a shame to miss it.
+
+As for this edition, the stakes were high with a new team taking over the reins of those who had come before them. 
+As a member of this new team, I had the pleasure to rise to the occasion and make sure the legacy stayed untarnished. 
+The new team did not disappoint, and even though there were minor hiccups during the day, in the end, the conclusion was clear. 
+A job well done, and a crowd that had gotten a day’s worth of interesting talks topped off with a barbecue blessed with early autumn sun.
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-10-24-ode-to-join/JOIN-01.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+Pieter Van den Wyngaert Opening the Conference in Style
+{: refdef}
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-10-24-ode-to-join/JOIN-02.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+Getting the inside scoop from our friends at AWS
+{: refdef}
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-10-24-ode-to-join/JOIN-03.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+Getting to Know your Fellow Conference Goers
+{: refdef}
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-10-24-ode-to-join/JOIN-04.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+Getting in Touch with the Serverless World (courtesy of November Five)
+{: refdef}
+
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2022-10-24-ode-to-join/JOIN-05.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+Ending the Conference by Queueing up for the Barbecue Diner
+{: refdef}
+
+Not only did we have a new team, but the founding member of the conference, the jWorks unit or Ordina was joined by their colleagues from the BOLD unit to make it an even more impressive package deal. 
+The talks reflected this, not only in a healthy mix of speakers but also in a healthy mix of topics ranging from the concerns about patching 3rd party software to the accessibility concerns of digital products for disabled people. 
+We had people from AWS talking about the various ways to modernize applications, and we had people from Flowable showing process automation at its finest. 
+With Flutter, experts came various insights into the world of mobile app design, and even the mysteries of the design system were laid bare for all to understand.
```

Comment:
'''With Flutter, experts came various insights''' this looks a bit strange to me, or does this need some rearranging of words?

Reply:
fixed